### DISCUSSION TITLE
Route erweitern funktioniert nicht (OpenRouteService)

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#103] [Q&A] [answered]
Ich habe eine Route von kurviger die läßt sich in cruiser normal öffnen. Aber wenn ich die Route erweitern möchte kommt die Meldung " Route nicht gefunden " ?

**Comment by devemux86** [comment]
Can you provide more details?
What router is selected in "Settings | Routing"?

Können Sie mehr Details angeben?
Welcher Router ist unter "Einstellungen | Routing" ausgewählt?

**Reply by 1maxi2** [reply]
OpenRouteService

**Comment by devemux86** [comment]
What navigation profile? What routing options?
Can you post the kurviger route and the extend point?

Partial route calculation now exists in navigation rerouting.
Partial route calculation in route planning is coming next.

Each router works differently, you may need to move some waypoints or select a different router, like OSRM.

---

Welches Navigationsprofil? Welche Routing-Optionen?
Können Sie die kurviger Route und den Verlängerungspunkt posten?

Die Teilroutenberechnung existiert jetzt in der Navigationsumleitung.
Als nächstes kommt die Teilroutenberechnung in der Routenplanung.

Jeder Router arbeitet anders, möglicherweise müssen Sie einige Wegpunkte verschieben oder einen anderen Router wie OSRM wählen.

**Reply by 1maxi2** [reply]
Habe jetzt festgestellt das es bei allen anderen kurviger Routen ganz normal funktioniert. 
Habe jetzt auf Graph Hopper umgestellt und dann funktioniert es auch mit dieser einen Route normal ?!
Sollte also kein allgemeines Problem sein. Danke für die Hilfe.

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

Did you send me the route? Thanks!

Looks like you have "Alternative routes" enabled?
OpenRouteService has a limitation of 100 km for the "Alternative" option: https://openrouteservice.org/restrictions

EDIT: it seems that the avoid options prevented this route calculation.

---

Haben Sie mir die Route geschickt? Danke!

Sieht so aus, als hätten Sie "Alternative Routen" aktiviert?
OpenRouteService hat eine Begrenzung von 100 km für die Option "Alternative": https://openrouteservice.org/restrictions

EDIT: es scheint, dass die Vermeidungsoptionen diese Routenberechnung verhindert haben.

**Reply by 1maxi2** [reply]
Hello
Habe alternative Route nicht aktiviert und konnte die Route 1x erweitern und beim 2. Mal steht wieder " Route nicht gefunden "
Ist aber für mich kein Problem wenn es mit Graph Hopper funktioniert 🙂

<img src="https://user-images.githubusercontent.com/124886589/228465600-addab259-ee37-42b5-9369-6b79353c74d3.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/124886589/228465617-cf52ffc9-d004-4cfe-9152-e27987746586.jpg" width="33%"/>

**Reply by devemux86** [reply]
Then it could be the avoid options?
You have  enabled the avoid motorways and toll roads.

Dann könnte es an den Vermeidungsoptionen liegen?
Sie haben die Option zum Vermeiden von Schnellstraßen und Mautstraßen aktiviert.

**Reply by 1maxi2** [reply]
👍 Habe die Vermeidung weggelassen und jetzt funktioniert es !

**Comment by 0709wiwiwi** [comment]
This has also been established many times in the Kurviger forum. 
If you set to avoid everything you don't get away from your hometown even.
